web/public/mgmt/config.yml

```
@@ -2173,6 +2173,15 @@ collections:
               - { label: Informational Message - Returned, name: informationalReturned, widget: string }
               - { label: Informational Message - Rejected, name: informationalRejected, widget: string }
               - { label: Informational Message - Incomplete, name: informationalIncomplete, widget: string }
+          - label: Elevator Violations Card
+            name: elevatorViolationsCard
+            collapsed: false
+            widget: object
+            fields:
+              - { label: Violation Notice Message Text, name: violationNoticeMessage, widget: string }
+              - { label: Violation Notice CTA Text, name: violationNoticeCTA, widget: string }
+              - { label: Violation Notice CTA Link, name: violationNoticeCTALink, widget: string }
```

[dust] it's not used everywhere, but there is a widget that will prevent the CMS user from inputing a space.
```suggestion
              - { label: Violation Notice CTA Link, name: violationNoticeCTALink, widget: nospace }
```